tests/realtikvtest/testkit.go

```diff
@@ -58,7 +58,7 @@ func RunTestMain(m *testing.M) {
 	})
 	tikv.EnableFailpoints()
 	opts := []goleak.Option{
-		goleak.IgnoreTopFunction("github.com/golang/glog.(*loggingT).flushDaemon"),
+		goleak.IgnoreTopFunction("github.com/golang/glog.(*fileSink).flushDaemon"),
```

Comment:
Maybe we should also ignore `google.golang.org/grpc/internal/grpcsync.NewCallbackSerializer` leak in `realtikvtest`.

ref https://github.com/pingcap/tidb/pull/51496

Reply:
it has been updated.